src/screens/DetailDomain/DetailDomainScreen.js

```diff
@@ -243,7 +243,7 @@ const DetailDomainScreen = (props) => {
 
   return (
     <View style={styles.container}>
-      <StatusBar translucent={false} />
+      <StatusBar translucent={false} barStyle={'light-content'} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `barStyle` prop of the `StatusBar` component has been set to `'light-content'`. This will change the color of the status bar content to light. If this is intended and matches with your app's theme across different screens, then it's fine. However, if you have a light background on this screen, this might make the status bar content hard to see. Please ensure that this change aligns with your design requirements.
```diff
- <StatusBar translucent={false} />
+ <StatusBar translucent={false} barStyle={'light-content'} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->